.circleci/config.yml

```diff
@@ -833,6 +833,66 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  check_production_gemfile:
+    parameters:
+      executor:
+        type: string
+        default: builder-ruby25
+    executor:
+      name: << parameters.executor >>
+      database: mysql
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "d8:d6:e4:53:b3:0e:22:76:01:e0:27:90:a6:c3:fa:df"
+      - checkout-with-submodules
+      - restore-gem-cache
+      - run:
+          name: Keep production gemfile up to date
+          command: |
+            echo "keeping production gemfiles up to date"
+            git config user.email "3scale@redhat.com"
+            git config user.name "3scale"
+            unset BUNDLE_FROZEN
+            bundle install --no-deployment
```

Comment:
I am not sure we need to install the gems. `bundle lock` should use the metadata from the gem spec.

Reply:
without this it failed because gems was installed with --deployment option

Reply:
Just use `BUNDLE_DEPLOYMENT` and `BUNDLE_FROZEN` to `"false"` ?
See https://github.com/rubygems/bundler/blob/master/spec/install/deploy_spec.rb#L297-L310